app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,12 @@
  *= require foundation_and_overrides
 
  */
+
+ body {
+  background: url("/background.png") no-repeat center fixed;
+  -moz-background-size: cover;
```

Comment:
Properties should be ordered background, background-color, -moz-background-size, -o-background-size, -webkit-background-size, background-size